src/openjd/sessions/_action_filter.py

```diff
@@ -32,6 +32,9 @@ class ActionMessageKind(Enum):
 )
 filter_matcher = re.compile(filter_regex)
 
+openjd_env_actions_filter_regex = "^(openjd_env|openjd_unset_env)+"
```

Comment:
```suggestion
openjd_env_actions_filter_regex = "^(openjd_env|openjd_unset_env)"
```

I think that we don't want the `+` here, right? That would match it one or more times.

Reply:
yes you are right, I will remove the `+`as  it's not required